pom.xml

```diff
@@ -52,6 +52,11 @@
       <artifactId>scala-library</artifactId>
       <version>2.11.7</version>
     </dependency>
+    <dependency>
+      <groupId>net.lingala.zip4j</groupId>
```

Comment:
A customer or the SWORD 1.3 API reported a problem that turned out to be related to his use of the lingala zip4j library. Here is a discussion topic about the problem that I found after doing a web-search:
http://www.lingala.net/zip4j/forum/index.php?topic=268.0
I'm not sure if zip4j is in the wrong, just mentioning it, so you know it may cause problems.